### ISSUE TITLE
JoinDesktop not working on Ubuntu 22.04

### ISSUE THREAD

**Issue by Gerd-St** [joaomgcd/JoinDesktop#62]
I have installed the com.joaomgcd.join_1.1.2_amd64.deb package. When starting the app it just opens an empty window for about 1 second and then closes.
Is there a workaround available?

**Comment by joaomgcd** [comment]
Thanks for the report. Are you please able to uninstall that version and try previous versions? Do those work? Thanks

**Comment by Gerd-St** [comment]
I just tried the AppImages for version 1.1.1 and 1.1.0. Identical behaviour with version 1.1.2 as both an AppImage and .deb package.

**Comment by joaomgcd** [comment]
Hmmm, never saw that issue before. Does "GPU process isn't usable" mean anything to you?

**Comment by joaomgcd** [comment]
Thanks. Does this maybe help? https://stackoverflow.com/questions/68874940/gpu-process-isnt-usable-goodbye

**Comment by Gerd-St** [comment]
Not really. I am running Ubuntu on a PC, not MacOS. And I have no idea on how I should replicate what is proposed there for running an AppImage or the .deb installation.

**Comment by Gerd-St** [comment]
I found a solution. I am using AppImageLauncher on my Ubuntu installation for easy starting the AppImages simiilar to other installed applications. After installing the Join AppImage a desktop file will be created at ./.local/shared/applications.
Within that desktop file I had to add the "--no-sandbox" option like this:

[Desktop Entry]
Name=Join Desktop (1.1.2)
Exec=/home/gerd/Applications/Join.Desktop-1.1.2_46b5ccfb3adf1171a3a0cca000ee71c8.AppImage --no-sandbox 
Terminal=false
Type=Application
Icon=appimagekit_3d9fa8b432c6b709c654a039a03ca4db_com.joaomgcd.join
StartupWMClass=Join Desktop
X-AppImage-Version=1.1.2
Comment=A companion app for the Join website
Categories=Utility;
.........

It may take a reboot after changing the file for taking effect.
